dco.ttl

```diff
@@ -790,84 +790,66 @@ dco:vesselName
       A name given to a research vessel. May not be unique.
       """^^rdf:XMLLiteral .
 
-##### Samples
-dco:GeoSample
+##### Repository
+dco:Repository
       a       owl:Class ;
-      rdfs:label "GeoSample"@en-US ;
-      rdfs:subClassOf vivo:InformationResource ;
+      rdfs:label "Repository"@en-US ;
+      rdfs:subClassOf dco:Object ;
       rdfs:subClassOf
               [ a       owl:Restriction ;
-                owl:allValuesFrom rdfs:Literal ;
-                owl:onProperty dco:hasElevation
+                owl:allValuesFrom dco:ResearchCommunity ;
+                owl:onProperty dco:associatedDCOCommunity
               ] ;
       rdfs:subClassOf
               [ a       owl:Restriction ;
-                owl:allValuesFrom rdfs:Literal ;
-                owl:onProperty dco:hasMaterial
+                owl:allValuesFrom core:GeographicLocation ;
```

Comment:
This doesn't seem quite right. Might also be vivo:Address or other geographic representations. Just not sure how to represent it here.